starboard/android/apk/app/src/main/java/dev/cobalt/media/MediaDrmBridge.java

```diff
@@ -373,6 +373,37 @@ private MediaDrmBridge(String keySystem, UUID schemeUUID, long nativeMediaDrmBri
     mSchemeUUID = schemeUUID;
     mMediaDrm = new MediaDrm(schemeUUID);
 
+    // Get info of hdcp connection, as 4k drm require hdcp 2.2
```

Comment:
Consider removing the mentioning of 4k and hdcp 2.2, as the code below isn't specific for this purpose.

Reply:
Fixed.